crates/wasmtime/src/engine/serialization.rs

```diff
@@ -194,6 +194,7 @@ impl Metadata {
             saturating_float_to_int: _,
             sign_extension: _,
             floats: _,
+            gc: _,
```

Comment:
Mind binding this as `gc` and asserting it's false down below (like `memory_control`)? We'll want to turn this on eventually and we won't want to forget to add handling here.